bin/zopen-install

```diff
@@ -573,8 +573,21 @@ if [ -z "${chosenRepos}" ]; then
     ph=$!
     killph="kill -HUP ${ph}"
     addCleanupTrapCmd "${killph}"
-    chosenRepos="$(${MYDIR}/zopen-query list --installed --no-header --no-versions)"
+    chosenRepos="$(${MYDIR}/zopen-query list --installed --no-header --no-version 2>&1)"
+    zqrc=$?
     ${killph} 2> /dev/null # if the timer is not running, the kill will fail
+    sleep 1 # give the above process time to clear
+    if [ ${zqrc} -ne 0 ]; then
+      printError "Query for installed packages unexpectedly failed; zopen-query returned message: '${chosenRepos}'"
+    fi
+  elif ${all}; then
```

Comment:
This logic was moved to https://github.com/ZOSOpenTools/meta/blob/77306255ad11ded06593cbda0d88a47e224e39e3/bin/zopen-install#L637. Do we still need it here?

Reply:
No - that was reinstated incorrectly